tests/mgbench/runners.py

```diff
@@ -121,22 +116,22 @@ def start_benchmark(self):
 
     def stop(self):
         ret, usage = self._cleanup()
-        assert ret == 0, "The database process exited with a non-zero " \
-            "status ({})!".format(ret)
+        assert ret == 0, "The database process exited with a non-zero " "status ({})!".format(ret)
         return usage
 
 
 class Client:
-    def __init__(self, client_binary, temporary_directory):
+    def __init__(self, client_binary, temporary_directory, username="", password=""):
```

Comment:
add typing library to all methods that you modify. 

`def __init__(self, client_binary: smth, temporary_directory: smth, username: str = "", password: str = ""):`